src/components/Property/Property.tsx

```diff
@@ -71,32 +72,38 @@ interface Props {
 }
 
 export const Property = memo(({ uri }: Props) => {
-  const description = useAppSelector(
-    (state) => state.properties.properties[uri]?.description
-  );
-
-  const value = useAppSelector((state) => state.properties.properties[uri]?.value);
-
+  const property = useGetProperty(uri);
   const setPropertyValue = useSubscribeToProperty(uri);
 
+  const description = property?.description;
+  const value = property?.value;
+
   if (!description || value === undefined) {
-    return null;
+    return <></>;
   }
 
   const ConcreteProperty = concreteProperties[description.type];
 
   if (!ConcreteProperty) {
-    // TODO: Bring back once all types are implemented
-    // console.error('Missing property type', property.description.type);
-    return null;
+    throw Error(`Missing property type: '${property.description.type}'`);
   }
 
-  // console.log(property);
+  // In most cases we want to show the label on top of the property input, but some types
+  // have a custom label built in in the input type. Keep track of which that is
+  const hasBuiltInLabel =
+    ConcreteProperty === BoolProperty || ConcreteProperty === TriggerProperty;
```

Comment:
These two properties have a built in label?

Reply:
Yes, here we don't want to show the property label on top of the input, the same way we do for the others.  Coult try to clarify that in the comment?

Reply:
Aha I see now what you mean! I think the comment is fine actually now that i now what you mean